install/src/command.rs

```diff
@@ -1283,3 +1283,29 @@ pub fn run(
         }
     }
 }
+
+pub fn list(config_file: &str) -> Result<(), String> {
+    let config = Config::load(config_file)?;
+
+    let entries = fs::read_dir(&config.releases_dir);
+    if entries.is_err() {
+        return Err("please make sure your config is set up properly".to_string());
+    }
```

Comment:
I think it'd still be useful to show the actual error. Also, we can use the `?` operator here so something like this (make sure you run `cargo fmt`, not sure if the indentation I provided is what the linter will want):
```suggestion
    let entries = fs::read_dir(&config.releases_dir)
        .map_err(|err| format!(
            "Failed to read install directory, double check that your configuration file is correct: {err}"
        ))?;
```